documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -225,14 +225,15 @@ curl -X POST \
   http://my-connect-cluster-connect-api:8083/connectors \
   -H 'Content-Type: application/json' \
   -d '{ "name": "my-source-connector",
-    "config":
+    "config": [
```

Comment:
Why do you think this should be an array?

Reply:
For different topics? Not sure now. I've dropped.